src/main/java/tasks/TaskList.java

```diff
@@ -0,0 +1,61 @@
+package tasks;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a list of Tasks object.
+ */
+public class TaskList {
+
+    ArrayList<Task> tasks;
```

Comment:
Perhaps you can include a visibility marker here.

Reply:
Sorry, what you mean by that?